central/postgres/report_pruning_test.go

```diff
@@ -61,9 +66,14 @@ func (s *ReportHistoryPruningSuite) SetupSuite() {
 	s.configDS = configDS
 	s.historyDS = historyDatastore.GetTestPostgresDataStore(s.T(), s.testDB)
 	s.blobDS = blobDatastore.NewTestDatastore(s.T(), s.testDB)
+	storageCollection := collectionPgStore.New(s.testDB.DB)
```

Comment:
[nit] Instead of doing this each time it's best to create a helper in the datastore.

You can either wait for @md2119 to merge his PR or you can use the code from his PR: https://github.com/stackrox/stackrox/pull/7397/files#diff-8720bc36b11ebb1268111dca9dc85b55bfbad21a1c329e3d0b0ce2a1550a557bR84

Then just do `collectionDatastore.GetTestPostgresDataStore(s.T(), s.testDB)`